addons/binding/pom.xml

```diff
@@ -31,6 +31,7 @@
     <module>org.openhab.binding.feed.test</module>
     <module>org.openhab.binding.folding</module>
     <module>org.openhab.binding.freebox</module>
+    <module>org.openhab.binding.gpio</module>
```

Comment:
The naming of the binding suggest generic gpio is that the case or is it only for raspberry pi? If not I saw some references to the pi. Note that gpio got really big with the pi but it was there already before the pi :-)